src/Tests/Topology/MetaModel/When_parsing_string_to_connection_string.cs

```diff
@@ -240,9 +239,8 @@ public void Should_equal_on_absence_of_sas_in_both()
         [Test]
         public void Should_not_equal_on_absence_of_sas_in_one()
         {
-            ConnectionStringInternal connectionString1, connectionString2;
-            ConnectionStringInternal.TryParse("Endpoint=sb://namespacename.servicebus.windows.net", out connectionString1);
-            ConnectionStringInternal.TryParse("Endpoint=sb://namespacename.servicebus.windows.net;SharedAccessKeyName=name;SharedAccessKey=key", out connectionString2);
+            ConnectionStringInternal.TryParse("Endpoint=sb://namespacename.servicebus.windows.net", out var connectionString1);
```

Comment:
Same as above

Reply:
commit comment has the reason https://github.com/Particular/NServiceBus.AzureServiceBus/pull/696/commits/494897bc309d4920d68ce2718bb2ba78e6b98e2a